src/snowflake/snowpark/mock/_functions.py

```diff
@@ -48,6 +52,30 @@
 _MOCK_FUNCTION_IMPLEMENTATION_MAP = {}
 
 
+class LocalTimezone:
```

Comment:
I'm not entirely satisfied with this solution. It might be better to have something like `ParameterEmulator` that holds this logic as well as logic for other session parameters.